app-android/app/src/main/java/de/tutao/tutanota/AndroidMobileSystemFacade.kt

```diff
@@ -168,4 +170,31 @@ class AndroidMobileSystemFacade(
 	override suspend fun openMailApp(query: String) {
 		Log.e(TAG, "Trying to open Tuta Mail from Tuta Mail")
 	}
+
+	private fun tryToLaunchStore() {
+		try {
+			startActivity(
+				activity,
+				Intent(Intent.ACTION_VIEW, Uri.parse("market://details?id=de.tutao.calendar")),
```

Comment:
we have package name as `BuildConfig` constant, we could use that

Reply:
I don't think so, the value inside BuildConfig will be the current compiled app package (```de.tutao.tutanota```) and we want to open ```de.tutao.calendar```

Reply:
We could do one of:

1. `activity.getString(R.string.package_name).replace("tutanota", "calendar")
2. Write the "sibling app" package id via `BuildConfig`